docs/src/use/migrate-to-9.0.0.md

```diff
@@ -130,3 +132,23 @@ ESLint v9.0.0 removes the deprecated `sourceCode.getComments()` method.
 **To address:** Replace with `sourceCode.getCommentsBefore()`, `sourceCode.getCommentsAfter()`, or `sourceCode.getCommentsInside()`.
 
 **Related Issues(s):** [#14744](https://github.com/eslint/eslint/issues/14744)
+
+## <a name="drop-function-style-rules"></a> Function-style rules are no longer supported
+
+ESLint v9.0.0 drops support for function-style rules. Function-style rules are rules created by exporting a function from a file rather than exporting an object with a `create()` method. This rule format was deprecated in 2016.
+
+**To address:** Update your rules to [the most recent rule format](../extend/custom-rules).
+
```

Comment:
It could be helpful to mention the lint rule that automatically fixes rules:

```suggestion

The [eslint-plugin/prefer-object-rule](https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/master/docs/rules/prefer-object-rule.md) rule can help enforce the usage of object-style rules and autofix any remaining funtion-style rules.

```

Reply:
> and autofix any remaining funtion-style rules

Awesome, thanks!